helper/iofs/iofs.go

```diff
@@ -0,0 +1,140 @@
+// Package iofs provides an adapter from billy.Filesystem to a the
+// standard library io.fs.FS interface.
+package iofs
+
+import (
+	"io"
+	"io/fs"
+	"path/filepath"
+
+	billyfs "github.com/go-git/go-billy/v5"
+	"github.com/go-git/go-billy/v5/helper/polyfill"
+)
+
+// Wrap adapts a billy.Filesystem to a io.fs.FS.
+func Wrap(fs billyfs.Basic) fs.FS {
```

Comment:
This would align this helper with all the other ones within `helper/`.

```suggestion
func New(fs billyfs.Basic) fs.FS {
```

Reply:
Done, including in the tests.